src/components/Tree/Tree.tsx

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactElement, useState } from "react";
+import Node, { TreeNode } from "./Node";
+import css from "./Tree.module.css";
+
+export interface TreeProps {
+    nodes: TreeNode[];
+    onSelect: (id: string) => void;
+    selectedNodeId?: string;
```

Comment:
why not using `activeNodeId` and rename it internally? this way it would follow the same "rule" as the slider and the dropdown of nadia

Reply:
49065e5